roofit/batchcompute/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ ROOT_LINKER_LIBRARY(RooBatchCompute
 target_include_directories(RooBatchCompute PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/res>)
 
 if(vdt OR builtin_vdt)
-  target_link_libraries(RooBatchCompute PUBLIC VDT)
+  target_link_libraries(RooBatchCompute PRIVATE VDT::VDT)
```

Comment:
This change from PUBLIC to PRIVATE breaks

https://github.com/root-project/root/pull/19348/files#diff-8923f606e55bc052901958c478616b498b030075d9dc3b285b4ee6dbc4663e20R20-R24

Because `RooBatchCompute_GENERIC` now no longer gets the VDT::VDT linked
leading to 
```
In file included from /build/jenkins/workspace/lcg_nightly_pipeline/build/projects/ROOT-HEAD/src/ROOT/HEAD/roofit/batchcompute/src/RooBatchCompute.cxx:23:
/build/jenkins/workspace/lcg_nightly_pipeline/build/projects/ROOT-HEAD/src/ROOT/HEAD/roofit/batchcompute/src/RooVDTHeaders.h:28:10: fatal error: vdt/cos.h: No such file or directory
   28 | #include <vdt/cos.h>
      |          ^~~~~~~~~~~
compilation terminated.
```

Reply:
OK, thanks for the notification!